src/persist-client/src/internal/machine.rs

```diff
@@ -1360,11 +1360,15 @@ pub mod datadriven {
         args: DirectiveArgs<'_>,
     ) -> Result<String, anyhow::Error> {
         let input = args.expect_str("input");
+        let stats = args.optional_str("stats");
```

Comment:
i found it a bit noisy when i tried it, but i will consider it! (i was realizing it would be nice to be able to opt in to just dumping out the entire stats data for a part also...)